packages/theme-chalk/src/menu.scss

```diff
@@ -250,9 +253,6 @@
 
   @include e(title) {
     @include menu-item;
-    padding-right: calc(
-      #{getCssVar('menu-base-level-padding')} + #{getCssVar('menu-icon-width')}
-    );
```

Comment:
Perhaps it should be disabled in collapse mode.

This PR will result in new style errors [Bug](https://element-plus.run/#eyJzcmMvQXBwLnZ1ZSI6IjxzY3JpcHQgc2V0dXA+XG5pbXBvcnQgeyByZWYsIHZlcnNpb24gYXMgdnVlVmVyc2lvbiB9IGZyb20gJ3Z1ZSdcbmltcG9ydCB7IHZlcnNpb24gYXMgRXBWZXJzaW9uIH0gZnJvbSAnZWxlbWVudC1wbHVzJ1xuaW1wb3J0IHsgRWxlbWVudFBsdXMsIExvY2F0aW9uIH0gZnJvbSAnQGVsZW1lbnQtcGx1cy9pY29ucy12dWUnXG5cbmNvbnN0IG1zZyA9IHJlZignSGVsbG8gV29ybGQhJylcbmNvbnN0IGlzQ29sbGFwc2UgPSByZWYoZmFsc2UpXG5jb25zdCBoYW5kbGVPcGVuID0gKGtleSwga2V5UGF0aCkgPT4ge1xuICBjb25zb2xlLmxvZyhrZXksIGtleVBhdGgpXG59XG5jb25zdCBoYW5kbGVDbG9zZSA9IChrZXksIGtleVBhdGgpID0+IHtcbiAgY29uc29sZS5sb2coa2V5LCBrZXlQYXRoKVxufVxuPC9zY3JpcHQ+XG5cbjx0ZW1wbGF0ZT5cbiAgPGVsLXJhZGlvLWdyb3VwIHYtbW9kZWw9XCJpc0NvbGxhcHNlXCIgc3R5bGU9XCJtYXJnaW4tYm90dG9tOiAyMHB4XCI+XG4gICAgPGVsLXJhZGlvLWJ1dHRvbiA6bGFiZWw9XCJmYWxzZVwiPmV4cGFuZDwvZWwtcmFkaW8tYnV0dG9uPlxuICAgIDxlbC1yYWRpby1idXR0b24gOmxhYmVsPVwidHJ1ZVwiPmNvbGxhcHNlPC9lbC1yYWRpby1idXR0b24+XG4gIDwvZWwtcmFkaW8tZ3JvdXA+XG4gICAgPGRpdiBjbGFzcz1cImNvbW1vbi1sYXlvdXRcIj5cbiAgICA8ZWwtY29udGFpbmVyPlxuICAgICAgPGVsLWFzaWRlIHN0eWxlPVwid2lkdGg6IGF1dG87XCI+XG4gICAgICA8ZWwtbWVudVxuICAgIGRlZmF1bHQtYWN0aXZlPVwiMlwiXG4gICAgOmNvbGxhcHNlPVwiaXNDb2xsYXBzZVwiXG4gICAgQG9wZW49XCJoYW5kbGVPcGVuXCJcbiAgICBAY2xvc2U9XCJoYW5kbGVDbG9zZVwiXG4gID5cbiAgICA8ZWwtc3ViLW1lbnUgaW5kZXg9XCIxXCI+XG4gICAgICA8dGVtcGxhdGUgI3RpdGxlPlxuICAgICAgICA8ZWwtaWNvbj48bG9jYXRpb24gLz48L2VsLWljb24+XG4gICAgICAgIDxzcGFuPk5hdmlnYXRvciBPbmU8L3NwYW4+XG4gICAgICA8L3RlbXBsYXRlPlxuICAgICAgPGVsLW1lbnUtaXRlbSBpbmRleD1cIjEtMVwiPml0ZW0gb25lPC9lbC1tZW51LWl0ZW0+XG4gICAgICA8ZWwtbWVudS1pdGVtIGluZGV4PVwiMS0yXCI+aXRlbSB0d288L2VsLW1lbnUtaXRlbT5cbiAgICAgIDxlbC1zdWItbWVudSBpbmRleD1cIjEtM1wiPlxuICAgICAgPHRlbXBsYXRlICN0aXRsZT5cbiAgICAgICAgPGVsLWljb24+PGxvY2F0aW9uIC8+PC9lbC1pY29uPlxuICAgICAgICA8c3Bhbj5OYXZpZ2F0b3IgT25lZnNhZWZnYXNnNWVzcmdyc3Ryc2hydGh0PC9zcGFuPlxuICAgICAgPC90ZW1wbGF0ZT5cbiAgICAgIDxlbC1tZW51LWl0ZW0gaW5kZXg9XCIxLTMtMVwiPml0ZW0gb25lPC9lbC1tZW51LWl0ZW0+XG4gICAgICA8ZWwtbWVudS1pdGVtIGluZGV4PVwiMS0zLTJcIj5pdGVtIHR3bzwvZWwtbWVudS1pdGVtPlxuICAgIDwvZWwtc3ViLW1lbnU+XG4gICAgPC9lbC1zdWItbWVudT5cbiAgICA8ZWwtc3ViLW1lbnUgaW5kZXg9XCIyXCI+XG4gICAgICA8dGVtcGxhdGUgI3RpdGxlPlxuICAgICAgICA8ZWwtaWNvbj48bG9jYXRpb24gLz48L2VsLWljb24+XG4gICAgICAgIDxzcGFuPk5hdmlnYXRvciBPbmU8L3NwYW4+XG4gICAgICA8L3RlbXBsYXRlPlxuICAgICAgPGVsLW1lbnUtaXRlbSBpbmRleD1cIjItMVwiPml0ZW0gb25lPC9lbC1tZW51LWl0ZW0+XG4gICAgICA8ZWwtbWVudS1pdGVtIGluZGV4PVwiMi0yXCI+aXRlbSB0d288L2VsLW1lbnUtaXRlbT5cbiAgICA8L2VsLXN1Yi1tZW51PlxuICA8L2VsLW1lbnU+XG4gICAgICA8L2VsLWFzaWRlPlxuICAgICAgPGVsLW1haW4+MTIzMTIzPC9lbC1tYWluPlxuICAgIDwvZWwtY29udGFpbmVyPlxuICA8L2Rpdj5cbjwvdGVtcGxhdGU+XG4iLCJzcmMvUGxheWdyb3VuZE1haW4udnVlIjoiPHNjcmlwdCBzZXR1cD5cbmltcG9ydCBBcHAgZnJvbSAnLi9BcHAudnVlJ1xuaW1wb3J0IHsgc2V0dXBFbGVtZW50UGx1cyB9IGZyb20gJy4vZWxlbWVudC1wbHVzLmpzJ1xuc2V0dXBFbGVtZW50UGx1cygpXG48L3NjcmlwdD5cblxuPHRlbXBsYXRlPlxuICA8QXBwIC8+XG48L3RlbXBsYXRlPlxuIiwiaW1wb3J0LW1hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge1xuICAgIFwiZWxlbWVudC1wbHVzXCI6IFwiaHR0cHM6Ly9wcmV2aWV3LTEzMTM1LWVsZW1lbnQtcGx1cy5zdXJnZS5zaC9idW5kbGUvaW5kZXguZnVsbC5taW4ubWpzXCIsXG4gICAgXCJlbGVtZW50LXBsdXMvXCI6IFwidW5zdXBwb3J0ZWRcIlxuICB9XG59IiwidHNjb25maWcuanNvbiI6IntcbiAgXCJjb21waWxlck9wdGlvbnNcIjoge1xuICAgIFwiYWxsb3dKc1wiOiB0cnVlLFxuICAgIFwiY2hlY2tKc1wiOiB0cnVlLFxuICAgIFwianN4XCI6IFwicHJlc2VydmVcIixcbiAgICBcInRhcmdldFwiOiBcIkVTTmV4dFwiLFxuICAgIFwibW9kdWxlXCI6IFwiRVNOZXh0XCIsXG4gICAgXCJtb2R1bGVSZXNvbHV0aW9uXCI6IFwiQnVuZGxlclwiLFxuICAgIFwiYWxsb3dJbXBvcnRpbmdUc0V4dGVuc2lvbnNcIjogdHJ1ZSxcbiAgICBcInR5cGVzXCI6IFtcImVsZW1lbnQtcGx1cy9nbG9iYWwuZC50c1wiXVxuICB9LFxuICBcInZ1ZUNvbXBpbGVyT3B0aW9uc1wiOiB7XG4gICAgXCJ0YXJnZXRcIjogMy4zXG4gIH1cbn1cbiIsInNyYy9lbGVtZW50LXBsdXMuanMiOiJpbXBvcnQgeyBnZXRDdXJyZW50SW5zdGFuY2UgfSBmcm9tICd2dWUnXG5pbXBvcnQgRWxlbWVudFBsdXMgZnJvbSAnZWxlbWVudC1wbHVzJ1xuXG5sZXQgaW5zdGFsbGVkID0gZmFsc2VcbmF3YWl0IGxvYWRTdHlsZSgpXG5cbmV4cG9ydCBmdW5jdGlvbiBzZXR1cEVsZW1lbnRQbHVzKCkge1xuICBpZiAoaW5zdGFsbGVkKSByZXR1cm5cbiAgY29uc3QgaW5zdGFuY2UgPSBnZXRDdXJyZW50SW5zdGFuY2UoKVxuICBpbnN0YW5jZS5hcHBDb250ZXh0LmFwcC51c2UoRWxlbWVudFBsdXMpXG4gIGluc3RhbGxlZCA9IHRydWVcbn1cblxuZXhwb3J0IGZ1bmN0aW9uIGxvYWRTdHlsZSgpIHtcbiAgcmV0dXJuIG5ldyBQcm9taXNlKChyZXNvbHZlLCByZWplY3QpID0+IHtcbiAgICBjb25zdCBsaW5rID0gZG9jdW1lbnQuY3JlYXRlRWxlbWVudCgnbGluaycpXG4gICAgbGluay5yZWwgPSAnc3R5bGVzaGVldCdcbiAgICBsaW5rLmhyZWYgPSAnaHR0cHM6Ly9wcmV2aWV3LTEzMTM1LWVsZW1lbnQtcGx1cy5zdXJnZS5zaC9idW5kbGUvaW5kZXguY3NzJ1xuICAgIGxpbmsuYWRkRXZlbnRMaXN0ZW5lcignbG9hZCcsIHJlc29sdmUpXG4gICAgbGluay5hZGRFdmVudExpc3RlbmVyKCdlcnJvcicsIHJlamVjdClcbiAgICBkb2N1bWVudC5ib2R5LmFwcGVuZChsaW5rKVxuICB9KVxufSIsIl9vIjp7InNob3dIaWRkZW4iOnRydWUsInN0eWxlU291cmNlIjoiaHR0cHM6Ly9wcmV2aWV3LTEzMTM1LWVsZW1lbnQtcGx1cy5zdXJnZS5zaC9idW5kbGUvaW5kZXguY3NzIn19)